amida-tech/indaba-client
# Feature/inba 555 move task

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?

Fixes a few display errors. Makes the "Complete Tasks" button move the workflow and fires an assessment copy command.

#### Related JIRA tickets:
There is backend PR that has the exact name as this. You will need it.

#### How should this be manually tested?
It is not necessary to blow away your database, but you will find checking and validation to be somewhat easier if you want to trace what happens on the backend.

Create a project with users, usergroups, a survey, stages and subjects. SUSUS. Be sure that the survey has some questions to it. For the purposes of this test, we'll say User1 and User2.

Assign a task to the first and second stages of any given subject, to User1 and User2 respectively. Logout of admin and log in as User1. Fill out some questions and hit the "Complete Task" button.

Verify you are returned to My Tasks and a toast informs you of your tasks completion.

Logout of User1. Login as User2. 
Go to My Tasks. Click on the task for stage 2.
Ensure that it is already populated with the same answers as those User1 chose. 


#### Background/Context

#### Screenshots (if appropriate):
